src/components/TodosList.js

```diff
@@ -0,0 +1,27 @@
+import TodoItem from "./TodoItem";
+const TodosList = ({ todosProps, handleChange, delTodo, setUpdate }) => {
+    // const delTodo = (id) => {
+    //     setTodos([
+    //       ...todos.filter((todo) => {
+    //         return todo.id !== id;
+    //       }),
+    //     ]);
+    //   };
```

Comment:
##  Commented out code
According to the best practices, leaving an unuse code as comment does not make the code clean. To make a clearn code , I suggest you to remove the commented out code.